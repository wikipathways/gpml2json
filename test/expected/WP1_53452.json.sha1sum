72c58bb66e749218b67678e3162f097d1678bcaa ?-
