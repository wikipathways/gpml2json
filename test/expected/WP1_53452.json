{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":[],"drawAs":"rect","gpmlElementName":"Pathway","height":0,"id":"hash://sha256/4e3b96e51fdfcf5c1c3d65a40c0d2a1fd11655d0df95833bd9222a59c48c2aee","kaavioType":"Group","name":"New Pathway","padding":5,"type":["Pathway"],"width":0,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","standardName":"New Pathway","displayName":"New Pathway","textContent":"Name: New Pathway","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=New%20Pathway&doSearch=1"},"entityMap":{}}
